packages/material-ui/src/ExpansionPanelSummary/ExpansionPanelSummary.js

```diff
@@ -134,7 +134,7 @@ const ExpansionPanelSummary = React.forwardRef(function ExpansionPanelSummary(pr
           })}
           edge="end"
           component="div"
-          tabIndex={-1}
+          tabIndex={null}
```

Comment:
Fix 
![Capture d’écran 2019-10-21 à 23 14 36](https://user-images.githubusercontent.com/3165635/67243755-996f0500-f458-11e9-8965-9835a81db20c.png)

Prior to this change, if you click the icon, the `space` key action didn't trigger the expansion panel.